Remotion/Web/Development.WebTesting.ControlObjects/TabbedMultiViewControlObject.cs

```diff
@@ -82,7 +83,8 @@ private WebTabStripTabDefinition ConvertToTabbedMultiViewTab ([NotNull] WebTabSt
           tabDefinition.ItemID.Substring (0, tabDefinition.ItemID.Length - "_Tab".Length),
           tabDefinition.Index,
           tabDefinition.Title,
-          tabDefinition.IsDisabled);
+          tabDefinition.IsDisabled,
+          HtmlTextWriterAttribute.Accesskey.ToString());
```

Comment:
Why the new parameter? Besides, I think you got something wrong, as this could be replaced with `"AccessKey"`.